fonts.css

```diff
@@ -0,0 +1,24 @@
+@font-face {
+  font-family: Montserrat, sans-serif;
+  src: url('fonts/Montserrat-Regular.woff') format('woff'),
+       url('fonts/Montserrat-Regular.woff2') format('woff2');
+  font-style: normal;
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: Montserrat, sans-serif;
+  src: url('fonts/Montserrat-Bold.woff') format('woff'),
+       url('fonts/Montserrat-Bold.woff2') format('woff2');
+  font-style: normal;
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: Montserrat, sans-serif;
+  src: url('fonts/Montserrat-Medium.woff') format('woff'),
+       url('fonts/Montserrat-Medium.woff2') format('woff2');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LOCO_proj_Boiko/pullRequest?prid=5568495)